src/api/test/functional/webui/spider_test.rb

```diff
@@ -25,6 +25,7 @@ def ignore_link?(link)
     return true if link.include?('/live_build_log/SourceprotectedProject')
     return true if link.include?('/live_build_log/home:Iggy/ToBeDeletedTestPack')
     return true if link.include?('/live_build_log')
+    return true if link.include?('/worker_capabilities')
```

Comment:
Hey @hennevogel appreciate the review! Spider was breaking on that link so I was thinking I should add an exception to it similar to the live_build_log link, but maybe there is a better way?

Reply:
Don't have it break? :)

Reply:
I think the other way to make this work without excluding the link would be to somehow create the data on the backend. I see some examples in `src/api/script/start_test_backend` but I don't know the path and data to provide the backend connection with. Is there any doc I can take a look to learn more?

Reply:
The backend is already started in the spider test, so this can't be it.

https://github.com/openSUSE/open-build-service/blob/master/src/api/test/functional/webui/spider_test.rb#L157